src/chi/components/dropdown/dropdown.scss

```diff
@@ -196,7 +196,7 @@
         padding-right: 3rem;
       }
 
-      &.-hasActive {
+      &.-has-active {
```

Comment:
Slight inconsistency with the `-label` class above. Here we've preserved the `-has` prefix but above we've removed. Can you verify this class is still needed? Not sure why it's applied to all parent `li` elements in the documentation menu:
![image](https://user-images.githubusercontent.com/130654/77351917-a475dc00-6cfb-11ea-909b-d316594a0252.png)

Reply:
Makes sense. After investigating I realized that -hasActive in doc menu is not necessary. Also, I decided to remove -has in -hasLabel because it was not a functional class. On the other hand, this -has-active has a functional purpose in tabbed navigation. We can discuss later what approach to take to fix it.